### DISCUSSION TITLE
Time zone with existing date/time field

### DISCUSSION THREAD

**Discussion by lanedsmu** [Budibase/budibase#10259] [Help] [answered]
Hi all,

I believe I understand that the "ignore time zone" option is available for some DB platforms but not others (and possibly [not always available for those we'd expect](https://github.com/Budibase/budibase/discussions/10060)?).

We have a lot of MSSQL tables with existing date/time data without timezone offset, and it appears Budibase always treats these as UTC.  Is there a short-term way for us to handle this without utilzing queries to add the offset?  We can do that, but that appears to limit some of the functionality and value offered by Budibase, vs utilizing tables straight-up.

Is there an issue or branch we can watch to get updates on when/if the ability to show date data without timezone calculations might be progressing?

Thanks much!  -Lane
![image](https://user-images.githubusercontent.com/80843079/231170933-7e6b7dba-4395-4e90-a7d0-31bb8b957f7c.png)

**Comment by melohagan** [comment, marked as answer]
Hey @lanedsmu 

I have created an issue here: https://github.com/Budibase/budibase/issues/10278
